packages/expo-camera/ios/EXCamera/EXCamera.m

```diff
@@ -709,6 +717,13 @@ - (void)startSession
       });
     }]];
 
+    [self.motionManager startAccelerometerUpdatesToQueue:[NSOperationQueue new]
```

Comment:
Perhaps it would be better to start and stop accelerometer updates to the queue only when the `responsiveOrientationWhenOrientationLocked` property is changed?